task_5_3.py

```diff
@@ -0,0 +1,27 @@
+from typing import Tuple
+
+
+def get_input() -> Tuple[float, float]:
+    while True:
+        try:
+            a = float(input('Enter a positive number a: '))
+            x = float(input('Enter any leading positive number x: '))
+            if a <= 0 or x <= 0:
+                raise ValueError
+            return a, x
+        except ValueError:
+            print("Invalid input, please enter a positive number.")
```

Comment:
you should create a function, which will validate one variable.
then you can apply it for all your variables in all labs